typhos/positioner.py

```diff
@@ -194,6 +219,30 @@ def stop(self):
         for device in self.devices:
             device.stop()
 
+    @QtCore.Slot()
+    def clear_error(self):
+        """
+        Clear the error messages from the device and screen.
+
+        The device may have errors in the IOC. These will be cleared by calling
+        the clear_error method.
+
+        The screen may have errors from the status of the last move. These will
+        be cleared from view.
+        """
+        for device in self.devices:
+            try:
+                device.clear_error()
```

Comment:
Thinking that there may be a `caput` behind `clear_error` that may block the GUI thread, I wonder if this should be done in the background?

Reply:
Good catch, this will freeze the gui if the network is being slow